README.md

```diff
@@ -86,5 +95,7 @@ The most common mood is :-)
 ```
 
 **think**: Should you use an array or a hash to solve this problem? Why?
+Because we were simply tallying the data and not trying to find specific days, I used an array to count the happy and sad days. I stored the dates of the happy and sad results instead of just tallying how many there were because it seemed like a useful piece of data to be able to access (which days were sad, which were happy, would be easy to access).
 
 **think**: What if we eventually want to add feelings to our analysis? Can we write this code in a way that will prevent us from having to re-write it later?
+I was thinking about storing the happy and sad arrays and their length in a hash of feelings, like { happy: happy.length, sad: sad.length}, and then doing a .sort_by method on the hash to find the most common one. That would be a refactor that would allow you to add more emotions with more ease.
```

Comment:
Yes! This is what I wanted. But then you don't need the array at all.

My overall_mood method, for reference:
```
def overall_mood(entries)
  return nil if entries.length == 0
  emoticons = Hash.new(0)
  entries.each do |entry|
    emoticon = analyze_mood(entry)
    emoticons[emoticon] += 1
  end
  return emoticons.max_by{|k,v| v}[0]
end
```

But I DO like your reasoning and why you did it.